src/amchart/CommonSerial.js

```diff
@@ -24,6 +23,10 @@
     CommonSerial.prototype.constructor = CommonSerial;
     CommonSerial.prototype._class += " amchart_CommonSerial";
 
+    CommonSerial.prototype.publish("xAxes", [], "widgetArray", "widgets", null, { max: 1, tags: ["Basic"] }); // max number of xAxes
+    CommonSerial.prototype.publish("yAxes", [], "widgetArray", "widgets", null, { tags: ["Basic"] });
+    //CommonSerial.prototype.publish("graphs", [""], "widgetArray", "widgets", null, { tags: ["Basic"] }); // this is breaking serialization why?
```

Comment:
It's possibly breaking serialization because "" isn't a widget. Did you try defaulting to [] ?